src/drive/web/modules/views/OnlyOffice/Toolbar/index.jsx

```diff
@@ -52,6 +54,7 @@ const Toolbar = () => {
 
   return (
     <>
+      <RealTimeQueries doctype={DOCTYPE_FILES} />
```

Comment:
Can you explain why it fixes the issue?

Reply:
L'éditeur Only Office est "autonome" à savoir qu'après avoir instancié l'app, le routeur de l'app redirige tout de suite sur l'éditeur. Il n'hérite donc d'aucun wrapper/provider de l'app.